bears/coffee_script/CoffeeLintBear.py

```diff
@@ -93,6 +93,8 @@ def generate_config(filename, file,
         """
         :param max_line_length:
             Maximum number of characters per line.
+            Set to infinite (MAX_SAFE_INTEGER in js) if max_line_length is
+            set to 0.
```

Comment:
Common parameters should have the same descriptions. So please change it to `When set to 0 allows infinite line length.` See https://github.com/coala/coala-bears/pull/2245/commits for more info.